pkg/controllermanager/controller/add.go

```diff
@@ -113,6 +114,12 @@ func AddToManager(ctx context.Context, mgr manager.Manager, cfg *controllermanag
 		return fmt.Errorf("failed adding SecretBinding controller: %w", err)
 	}
 
+	if err := (&shootstate.Reconciler{
+		Config: *cfg.Controllers.ShootState,
+	}).AddToManager(mgr); err != nil {
+		return fmt.Errorf("failed adding ShootState controller : %w", err)
```

Comment:
Cosmetic nit:

```suggestion
		return fmt.Errorf("failed adding ShootState controller: %w", err)
```

Reply:
Thanks for catching this trailing space. Removed it in my latest updates.